.golangci.yml

```diff
@@ -20,7 +32,7 @@ linters-settings:
   maligned:
     suggest-new: true
   dupl:
-    threshold: 100
+    threshold: 150
```

Comment:
Given that this is the default, maybe it is better to just remove it ?

Reply:
Ah, missed the inline comment. See https://github.com/loadimpact/k6/pull/976#issuecomment-476119734 for my reasoning